packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -225,6 +225,7 @@ export class BlobManager extends TypedEventEmitter<IBlobManagerEvents> {
 		// Read the feature flag that tells whether to throw when a tombstone blob is requested.
 		this.throwOnTombstoneLoad =
 			this.mc.config.getBoolean(throwOnTombstoneLoadKey) === true &&
+			!(this.runtime as ContainerRuntime).disableGcTombstoneEnforcement &&
```

Comment:
Is modifying IBlobManagerRuntime to include the disableGCTombstoneEnforcement flag possible/relatively easy instead of importing ContainerRuntime? Somehow, I have a feeling there's going to be a circular dependency here.

Reply:
I'll check

Reply:
No circular dependency since it's all in the same pacakge.